app/models/volunteer.rb

```diff
@@ -58,18 +60,40 @@ class Volunteer < ApplicationRecord
 
   default_scope { order(created_at: :desc) }
 
-  scope :seeking_clients, (-> { where(state: SEEKING_CLIENTS) })
-
   scope :created_between, ->(start_date, end_date) { where(created_at: start_date..end_date) }
 
+  scope :with_hours, (-> { joins(:hours).distinct })
+
+  scope :all_accepted, (-> { where(acceptance: :accepted) })
+  scope :all_resigned, (-> { where(acceptance: :resigned) })
+  scope :all_rejected, (-> { where(acceptance: :rejected) })
+  scope :all_undecided, (-> { where(acceptance: :undecided) })
```

Comment:
I think the enum already adds scopes for these? (without the `all_`)

Reply:
right! 😉